### ISSUE TITLE
config tests fails locally on macOS likely due to platform specific `stat` ouput

### ISSUE THREAD

**Issue by ashleygwilliams** [cloudflare/wrangler#372]
```
test it_generates_the_config ... FAILED

failures:

---- it_generates_the_config stdout ----
thread 'it_generates_the_config' panicked at 'assertion failed: out.starts_with("600")', tests/config.rs:45:9
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.


failures:
    it_generates_the_config

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out

error: test failed, to rerun pass '--test config'
```

**Comment by ashleygwilliams** [comment]
confirmed. stat has no `-c` option on mac.

```
out = Ok(Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "stat: illegal option -- c\nusage: stat [-FlLnqrsx] [-f format] [-t timefmt] [file ...]\n" })
thread 'it_generates_the_config' panicked at 'assertion failed: false', tests/config.rs:48:9
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
test it_generates_the_config ... FAILED
```

**Comment by xtuc** [comment]
My bad sorry. I'm not using MacOS, maybe we should have a CI running it?

**Comment by ashleygwilliams** [comment]
you're all good @xtuc i ended up setting to only run on linux, which it does on our CI :) forgot to close this yesterday! solved in #365
